CHANGES.md

```diff
@@ -28,6 +28,7 @@
 - Remove duplicate items in class and class types (#105, @azzsal)
 - Fixed loading of modules whose signature is given by a path to a module type:
   `module X : Y` (#128, @panglesd)
+- Fixed command line flags associated with displaying the diff (#136, @azzsal)
```

Comment:
This would have to go under Changed as we're not fixing a bug here, but modifying our interface and how we use existing CLI flags and options.

Since the original change has not been released yet I would merge this with the following entry:
```markdown
- Add word-level display of textual diffs (#131, @azzsal)
```
turning it into:
```markdown
- Add word-level display of textual diffs in `api-diff`, enabled with the `--word-diff` flag  (#131, #136, @azzsal)
- Add `--plain` flag to `api-diff` to use text markers for inline highlighting. Can be used when the output does not support colors (#136, @azzsal)
```

A changelog is meant to be read by users not only developers of the project, we need to give context and actionable info there.